src/devtools/client/debugger/src/components/Editor/Breakpoints/Breakpoint.tsx

```diff
@@ -98,6 +98,7 @@ class Breakpoint extends PureComponent<BreakpointProps> {
     const doc = getDocument(sourceId);
 
     resizeBreakpointGutter(editor.codeMirror);
+    // @ts-expect-error method doesn't exist on Doc
```

Comment:
This feels like a good sign that we have this typed incorrectly 🤔 . Is it supposed to be `Editor`?